build-tools/packages/build-cli/src/commands/check/layers.ts

```diff
@@ -0,0 +1,73 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import path from "path";
+import { Flags } from '@oclif/core';
+import { LayerGraph, Timer, writeFileAsync } from "@fluidframework/build-tools";
+import { BaseCommand } from "../../base";
+
+// eslint-disable-next-line @typescript-eslint/no-inferrable-types
+const packagesMdFileName: string = "PACKAGES.md";
+
+export class LayerCheck extends BaseCommand {
+  static description = 'Make sure the dependencies between Fluid Framework packages are properly layered';
+
+  static flags = {
+    md: Flags.string({
```

Comment:
```suggestion
    md: Flags.directory({
```

Did you try this? It might not work since we might not expect a "real path". In that case is there any validation you could/should do in the flag's `parse` function?

Reply:
The only validation I did here is for the default path: `./bin/dev check layers --md=.`


![image](https://user-images.githubusercontent.com/46719950/181350799-28db8461-a439-4b2d-a2e1-494b0129394b.png)

Reply:
moved PACKAGES.md to a folder inside build-cli and tested the md flag by passing this path: 

![image](https://user-images.githubusercontent.com/46719950/181384179-2611db39-4dc3-4fcb-b8b6-6f12fda66ae1.png)